### PR TITLE
SIP-397-Sepolia-Fjord

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

**Review by leomassazza — APPROVED** [review]

File: omnibus-base-mainnet-andromeda.toml [hunk 1]
```diff
@@ -3,20 +3,20 @@ version = "41"
 description = "Andromeda deployment"
 preset = "andromeda"
 include = [
-    "tomls/core.toml",                                              # Provision the core system
```
Comment (noisekit) [inline omnibus-base-mainnet-andromeda.toml:6]: Need version bump

File: tomls/omnibus-base-sepolia-andromeda/perps/perps-keeper-cost.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+[provision.perps_gas_oracle_node]
+source = "<%= settings.perps_keeper_cost_package %>"
+targetPreset = "<%= settings.target_preset %>"
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/perps-keeper-cost.toml:3]: I think the new convention is to use "target" with full name not just targetPreset, not a blocker tho

File: omnibus-base-mainnet-andromeda.toml (hunk 1 above)
Comment (kaleb-keny) [reply omnibus-base-mainnet-andromeda.toml:6]: mainnet isn't changed in this PR

File: omnibus-base-mainnet-andromeda.toml [hunk 3]
```diff
@@ -3,20 +3,20 @@ version = "41"
...
     "tomls/markets/common/bigcap-settings.toml",
-    "tomls/omnibus-base-mainnet-andromeda/spot/usdc.toml",          # sUSDC spot market/wrapper
-    "tomls/permit-all-perps-perpsHighcapSystem.toml",               # Enable perps market
+    "tomls/omnibus-base-mainnet-andromeda/spot/usdc.toml",               # sUSDC spot market/wrapper
+    "tomls/permit-all-perps-perpsHighcapSystem.toml",                    # Enable perps market
     "tomls/permit-pdao-createMarket.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/referrers.toml",
     "tomls/omnibus-base-mainnet-andromeda/perps/global.toml",
-    "tomls/oracles/perps-keeper-cost.toml",                         # Add gas oracle for keeper fees
+    "tomls/omnibus-base-mainnet-andromeda/perps/perps-keeper-cost.toml", # Add gas oracle for keeper fees
```
Comment (noisekit) [inline omnibus-base-mainnet-andromeda.toml:19]: Can we rollback this file for now? This is base sepolia pr

File: omnibus-base-mainnet-andromeda.toml (hunk 1 above)
Comment (noisekit) [reply omnibus-base-mainnet-andromeda.toml:6]: It has been. Lets rollback this file please

Comment (noisekit) [reply omnibus-base-mainnet-andromeda.toml:6]: Ah ok I see. No problem

File: omnibus-base-mainnet-andromeda.toml (hunk 3 above)
Comment (kaleb-keny) [reply omnibus-base-mainnet-andromeda.toml:19]: There will be a different PR for updating mainnet, right after PR gets sent to the chain.

File: omnibus-base-mainnet-andromeda.toml (hunk 1 above)
Comment (noisekit) [reply omnibus-base-mainnet-andromeda.toml:6]: Yeah. Checked the build log - no changes indeed

**Review by noisekit — APPROVED** [review]

**Pull request merged**